storage/innobase/xtrabackup/test/t/alter_redo.sh

```diff
@@ -0,0 +1,93 @@
+########################################################################
+#test redo enable/disable feature with backup and incremental backup	
+########################################################################
+. inc/common.sh
+require_server_version_higher_than 8.0.20
+start_server 
+
+vlog "### case #1 check when redo log is disabled ###"
+mysql -e "alter instance disable innodb redo_log"
+run_cmd_expect_failure xtrabackup --backup \
+        --target-dir=$topdir/backup  2>&1 \
+        | tee $topdir/pxb.log
+grep "Redo log is disabled can not take consistent backup" $topdir/pxb.log || die "missing error message"
```

Comment:
Add another case,  redo is disabled and enabled before full backup.

Also for these scenarios, please do DMLs just after disabling